src/dependabot/update_metadata.ts

```diff
@@ -64,7 +68,7 @@ export async function parse (commitMessage: string, body: string, branchName: st
           newVersion: nextVersion,
           compatScore: await scoreFn(dependency['dependency-name'], lastVersion, nextVersion, chunks[1]),
           maintainerChanges: newMaintainer,
-          dependencyGroup: dependencyGroup,
+          dependencyGroup,
```

Comment:
I think we have a weird linter rule that will tell you to do this? Let's ignore it for this PR
```suggestion
          dependencyGroup: dependencyGroup,
```

Reply:
I will undo this change then

Reply:
I know I had asked you to change this, but I was wrong 😭 
You had it right the first time. Could you change it back and remove the eslint exception?

Reply:
This ESLint rule has been fixed now